image-files/first-boot-centos.sh

```diff
@@ -4,6 +4,15 @@ cd $KUBEVIRT_ANSIBLE_DIR
 echo "[masters]" >> inventory-aws
 hostname >> inventory-aws
 
+# make sure we use a weave network that doesnt conflict
+for num in `seq 30 50` ; do
+ ip r | grep -q 172.$num
+ if [ "$$" != "0"] ; then
+  sed -i "s/172.30/172.$num/" roles/kubernetes-master/vars/main.yml
```

Comment:
There is an issue with symbolic links being not preserved when the kubevirt-ansible directory is uploaded to the image. Note when we execute the playbooks in kubevirt-ansible/playbooks it uses the roles from this symlink: kubevirt-ansible/playbooks/roles. Because the symbolic links weren't being preserved we produced a copy of the roles directory under kubevirt-ansible/playbooks/roles. I'll put together a patch to fix this.

Reply:
This means that although we edit the file roles/kubernetes-master/vars/main.yml here, it isn't the one that is actually used. The one actually used is playbooks/roles/kubernetes-master/vars/main.yml.
https://github.com/kubevirt/kubevirt-ansible/blob/master/playbooks/roles